### ISSUE TITLE
Official support?

### ISSUE THREAD

**Issue by molotch** [dougmsft/microsoft-avro#21]
Is this library officially supported and actively maintained? We're looking into using Event Hubs capture, the lack of actively maintained Avro libraries for .NET makes that a hard sell.

**Comment by m1nkeh** [comment]
yeah, i second this comment.. the .Net Avro landscape is a muddle right now..

**Comment by igitur** [comment]
@dougmsft If this repo isn't maintained anymore, please archive it.
